routing/localchans/manager.go

```diff
@@ -182,9 +192,14 @@ func (r *Manager) updateEdge(tx kvdb.RTx, chanPoint wire.OutPoint,
 		newSchema.FeeRate,
 	)
 
-	inboundFee := newSchema.InboundFee.ToWire()
-	if err := edge.ExtraOpaqueData.PackRecords(&inboundFee); err != nil {
-		return err
+	if !newSchema.InboundFee.IsNone() {
```

Comment:
Should use `WhenSome` instead

Reply:
Yeah we want to remove `UnsafeFromSome`. 

You can also write it like this (to catch that error):
```
errr := fn.MapOptionZ(newSchema.InboundFee, func(f models.InboundFee) error {
      return edge.ExtraOpaqueData.PackRecords(&inboundWireFee)
})
```

But see my other comment, I think we should just have the new fields be a direct part of the struct instead of encoding into the extra data. Though I can understand if you leave to wish it like this, as your main goal was resolving the bug re default values.